chart/values.yaml

```diff
@@ -0,0 +1,233 @@
+  # - -v=8
+  # - -varnish-additional-parameters=vcc_allow_inline_c=on
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  enabled: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+rbac:
+  enabled: true
+
+podSecurityPolicy:
+  enabled: false
+  # name: unrestricted-psp
+
+podAnnotations: {}
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext: {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+
+initContainers: {}
+# initContainers: |
+#   - args:
+#     - -c
+#     - |
+#       echo "Copying external varnish template from..."
+#     command:
+#     - sh
+#     image: busybox:latest
+#     imagePullPolicy: IfNotPresent
+#     name: varnishtemplate
+#     resources: {}
+#     terminationMessagePath: /dev/termination-log
+#     terminationMessagePolicy: File
+#     volumeMounts:
+#     - name: template
+#       mountPath: /etc/varnish/tmpl
+#
+extraVolumes: {}
+# extraVolumes:
+#   - emptyDir: {}
+#     name: template
+
+extraMounts: {}
+# extraMounts:
+#   - name: geoip
+#     mountPath: /var/lib/geoip
+
+service:
+  type: ClusterIP
+  port: 80
+  target: 80
+
+ingress:
+  enabled: false
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  hosts: []
+  # hosts:
+  #   - host: www.example.com
+  #     paths:
+  #       - path: /
+  #         pathType: Prefix
+  #         backend:
+  #           service:
+  #             name: kube-httpcache
+  #             port:
+  #               number: 80
+  #       - path: /backend
+  #         backend:
+  #           name: backend-service
+  #           port:
+  #             number: 8080
+  #   - host: www2.example.com
+  #     paths:
+  #       - path: /
+  #         pathType: Prefix
+  #         backend:
+  #           name: kube-httpcache
+  #           port:
+  #             number: 80
+  tls: []
+  #  - secretName: chart-example-tls
+  #    hosts:
+  #      - chart-example.local
+
+resources: {}
+  # We usually recommend not to specify default resources and to leave this as a conscious
+  # choice for the user. This also increases chances charts run on environments with little
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  # limits:
+  #   cpu: 100m
+  #   memory: 128Mi
+  # requests:
+  #   cpu: 100m
+  #   memory: 128Mi
+
+autoscaling:
+  enabled: false
+  minReplicas: 1
+  maxReplicas: 100
+  targetCPUUtilizationPercentage: 80
+  # targetMemoryUtilizationPercentage: 80
+
+nodeSelector: {}
+
+tolerations: []
+
+affinity: {}
+
+livenessProbe: {}
+# livenessProbe:
+#   httpGet:
+#    path: /
+#    port: 6083
+readinessProbe: {}
```

Comment:
Some basic default readiness/lieness Probes would be may a good idea too (e.g. tcp socket check), better then nothing?

Reply:
Created #57 as follow-up issue